src/Interpreters/RewriteUniqToCountVisitor.cpp

```diff
@@ -22,8 +22,8 @@ namespace
 bool matchFnUniq(String func_name)
 {
     auto name = Poco::toLower(func_name);
-    return name == "uniq" || name == "uniqHLL12" || name == "uniqExact" || name == "uniqTheta" || name == "uniqCombined"
-        || name == "uniqCombined64";
+    return name == "uniq" || name == "uniqhll12" || name == "uniqexact" || name == "uniqtheta" || name == "uniqcombined"
```

Comment:
I figured out that this is theoretically wrong.

The function `uniqCombined` is not case-insensitive.
A user could register a user-defined function with `uniqcombined` name, and the optimization will be applied incorrectly.